boards/cats_rev1/Core/Src/control/data_processing.c

```diff
@@ -5,80 +5,76 @@
 #include "control/data_processing.h"
 #include <string.h>
 
-// Utility function to find minimum of two integers
-int min(int x, int y) { return (x < y) ? x : y; }
+#define SWAP(a,b) { float temp=(a);(a)=(b);(b)=temp; }
```

Comment:
You can remove the changes for cats_rev1, this is deprecated now. The cats_rev1Pro is being actively worked on